### DISCUSSION TITLE
Freeplay Favorite option

### DISCUSSION THREAD

**Discussion by RexDX7** [Snirozu/Funkin-Psych-Online#87] [Ideas]
just like on V-Slice, i would love to be able to select specific songs as my favorites using a button ("F" for example), then with another button (Maybe "CTRL") show ONLY the favorited songs, favorited songs would appear with a heart like on V-Slice

**Comment by fnfbf2000** [comment]
yeah, good idea

**Comment by Glitchboy911** [comment]
how do i get my freeplay songs back i cant play them anymore and i dont know how to get them back

**Reply by TechnikTil** [reply]
press alt+r or whatever your reset button is
